### DISCUSSION TITLE
Other System Information

### DISCUSSION THREAD

**Discussion by ghost** [srsran/srsRAN_Project#291] [General Help]
Before I go down the rabbit hole of attempting this - and unsure if it is supported at this point... 

Has anyone had success with implementing other System Information (OSI) blocks with NR? SIB2, SIB3, SIB4, SIB5 specifically...

I see the configuration reference includes SIB19 but i assume within the code there are no definitions for OSI and therefore it is not capable in the current release? 

Any information would be helpful - thank you.

**Comment by herlesupreeth** [comment]
> I see the configuration reference includes SIB19 but i assume within the code there are no definitions for OSI and therefore it is not capable in the current release?

You are right. Scheduling any other System Information other than SIB1 is not implemented. Apart from scheduling I think configuration aspects of SIB19 is implemented

**Comment by pgawlowicz** [comment]
Scheduling of OSI was implemented and most probably it will be available in the next release.
